File: Pattern Questions/Pattern4.cpp
/*Solution for
1
2 2
3 3 3
4 4 4 4
5 5 5 5 5
*/

#include <bits/stdc++.h>
using namespace std;

void printTriangle(int n)
{
    for (int i = 1; i != n + 1; i++)
    {
        for (int j = 1; j != i + 1; j++)
        {
            cout << i << ' ';
        }
        cout << endl;
    }
}

int main()
{
    printTriangle(5);
    return 0;
}

File: Pattern Questions/Pattern4.java
public class Pattern4 {
    public static void main(String[] args) {
        
    }
    public static void printTriangle(int n){
        for(int i=1;i!=n+1;i++){
            for(int j=1;j!=i+1;j++){
                System.out.print(i);
                System.out.print(' ');
            }
            System.out.println();
        }
    }
}
